src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -19,6 +48,19 @@
     <a href="/activity.jsp">Activity Feed</a>
   </nav>
 
+<div id="feed">
+  <ul>
+    <%
+      for (Conversation convo : conversations) {
```

Comment:
nit:

(Conversation conversation : conversations)

'convo' is not a well-known abbreviation for conversation (as far as I know), so naming that various 'conversation' is better, more clear, more readable.